src/main/resources/static/js/main.js

```diff
@@ -75,26 +75,33 @@ document.addEventListener('DOMContentLoaded', () => {
 
     forms.forEach(item => item.addEventListener('submit', function (e) {
         e.preventDefault();
-
-        let formData = new FormData(this);
-        formData = Object.fromEntries(formData);
-        console.log(formData)
-        ajaxSend(formData);
-        this.reset();
+        var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
```

Comment:
I really not sure about it. But probably it's better to `us` let instead of `var`. What do think?

Reply:
Fixed